moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_optimizer.h

```diff
@@ -61,7 +61,11 @@ class ChompOptimizer
 
   virtual ~ChompOptimizer();
 
-  void optimize();
+  /**
+   * this function optimizes the chomp cost function and tries to find an optimal path
```

Comment:
Remove "this function" (more concise) and capitalize Optimizes

Reply:
chomp -> CHOMP

Reply:
done